src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -8,7 +8,7 @@
 * (Deliberately contains bugs.)
 */
 public class TorpedoStore {
-
+  Random generator = new Random();
```

Comment:
This name is not sufficient in Clean Code standards! The name should be more descriptive, such as `randomNumberGenerator` or similar.